src/FwdState.cc

```diff
@@ -75,7 +75,7 @@ static OBJH fwdStats;
 #define MAX_FWD_STATS_IDX 9
 static int FwdReplyCodes[MAX_FWD_STATS_IDX + 1][Http::scInvalidHeader + 1];
 
-static PconnPool *fwdPconnPool = new PconnPool("server-peers", NULL);
+PconnPool *fwdPconnPool = new PconnPool("server-peers", NULL);
```

Comment:
auto and, hopefully, const

Reply:
The compiler does not allow me to use auto, it produces a type conflict with the "extern PconnPool *fwdPconnPool" declaration inside FwdState.h file.
The const also can not be used here.